CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_netsuite v0.9.0
```

Comment:
I would also add a test update section that highlights your addition of `account_id` in the `netsuite2__balance_sheet` model test

Reply:
added!